aptos-move/move-examples/mint_nft_v2/3-Adding-Admin/sources/create_nft_with_resource_and_admin_accounts.move

```diff
@@ -0,0 +1,201 @@
+module mint_nft_v2_part3::create_nft_with_resource_and_admin_accounts {
+    use std::bcs;
+    use std::error;
+    use std::signer;
+    use std::object;
+    use std::string::{Self, String};
+    use std::timestamp;
+    use std::vector;
+    use std::table::{Self, Table};
+    use aptos_framework::account::{Self, SignerCapability};
+
+    use aptos_token_objects::aptos_token::{Self, AptosToken};
+
+    // This struct stores an NFT collection's relevant information
+    struct MintConfiguration has key {
+        signer_capability: SignerCapability,
+        collection_name: String,
+        token_name: String,
+        token_uri: String,
+        whitelist: Table<address, bool>,
```

Comment:
Let's use smart_table here instead for better gas and storage efficiency

Reply:
@xbtmatt